dubbo-serialization/dubbo-serialization-gson/main/java/org/apache/dubbo/common/serialize/gson/GsonSerialization.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.dubbo.common.serialize.gson;
+
+import org.apache.dubbo.common.URL;
+import org.apache.dubbo.common.serialize.ObjectInput;
+import org.apache.dubbo.common.serialize.ObjectOutput;
+import org.apache.dubbo.common.serialize.Serialization;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+/**
+ * Created by cr on 2018/10/23.
+ */
+public class GsonSerialization   implements Serialization {
+
+
+    @Override
+    public byte getContentTypeId() {
+        return 26;
+    }
+
+    @Override
+    public String getContentType() {
+        return "text/json";
```

Comment:
seems it should be `gson` instead of `json`?

Reply:
already fix   "text/gson"    , why this  commit  build failed???         please tell me